site/src/docs/getting-started/installation/index.md

```diff
@@ -69,4 +69,4 @@ Put the next code snippet on a page of your site where you want to have comments
 
 After that widget will be rendered inside this node.
 
-If you want to set this up on a Single Page App, see [appropriate doc page](https://github.com/umputun/remark42/blob/master/docs/spa.md).
+If you want to set this up on a Single Page App, see [appropriate doc page](https://github.com/umputun/remark42/blob/master/site/src/docs/getting-started/installation/index.md).
```

Comment:
As I can see docs on SPA usage is placed in `site/src/docs/configuration/frontend/index.md`